Brokerages/Fxcm/FxcmBrokerage.Util.cs

```diff
@@ -224,7 +184,8 @@ private static DateTime FromJavaDate(java.util.Date javaDate)
                                 cal.get(java.util.Calendar.DAY_OF_MONTH),
                                 cal.get(java.util.Calendar.HOUR_OF_DAY),
                                 cal.get(java.util.Calendar.MINUTE),
-                                cal.get(java.util.Calendar.SECOND));
+                                cal.get(java.util.Calendar.SECOND),
+                                cal.get(java.util.Calendar.MILLISECOND));
```

Comment:
Please keep PRs to one issue. No worries on this pull, but this bug fix should have been in a separate PR.